src/components/TxData.js

```diff
@@ -93,6 +96,26 @@ class TxData extends React.Component {
     });
   }
 
+  fetchWalletAddressesMap = async () => {
+    const inputs = this.props.transaction.inputs;
+    const outputs = this.props.transaction.outputs;
+
+    const addresses = [
+      ...inputs.map((input) => input.decoded.address),
+      ...outputs.map((output) => output.decoded.address),
```

Comment:
When we have a data script output (we don't have addresses on it), will this throw an error?

Reply:
It would, luckily I fixed it in a73171da5314901a014f1e417bb497f45e3779dd